config.yaml

```diff
@@ -39,17 +39,17 @@ metamorph:
   log:
     file: ./data/metamorph.log # location of log file
   db:
-    mode: badger # db mode inidicates which db to use. Value can be one of badger | postgres
+    mode: badger # db mode inidicates which db to use. Value can be one of badger | postgres | dynamodb
     postgres: # postgres db configuration in case that mode: postgres
       host: localhost
       port: 5432
       name: metamorph
       user: arc
       password: arc
     dynamodb:
-      aws_access_key_id:
-      aws_secret_access_key:
-      aws_session_token:
+      awsAccessKeyID:
```

Comment:
This cannon be changed only here. It also needs to be changed where it is used by viper: https://github.com/bitcoin-sv/arc/blob/76bf601b0e02f1403fc1bab0c480442b9e8a0563/metamorph/store/dynamodb/dynamodb.go#L32